<launch>
  <arg name="enableEvologicsDriver"   default="true"/>
  <arg name="simulateGps"             default="false"/>

  <!-- Evologics driver -->
  <group if="$(arg enableEvologicsDriver)">
    <rosparam command="load" file="$(find turbot)/config/evologics_gs.yaml"/>
    <param name="/evologics/config_srv_dir" value="$(find turbot)/config/evologics_services.yaml"/>
    <node name="evologics_ros_sync_gs" pkg="evologics_ros_sync" type="evologics_ros_gs.py" output="screen"/>
  </group>


  <!-- USBL POSITION -->
  <node pkg="usbl_position" name="usbl_position" type="usbl_position" output="screen">
    <remap from="/usbl_position/modem_delayed" to="/sensors/modem_delayed"/>
    <remap from="/usbl_position/buoy_ned" to="/sensors/gps_buoy_emlid_ned"/>
    <remap from="/sensors/buoy" to="/sensors/gps_buoy_emlid_filtered"/>
  </node>

</launch>
